src/main.spec.ts

```diff
@@ -1,5 +1,5 @@
-describe("This test", function() {
-  it("will always pass", function() {
+describe('This test', function() {
```

Comment:
could you add a comment to this so people know why this is here and that it should not be removed